homeassistant/components/tautulli/config_flow.py

```diff
@@ -0,0 +1,140 @@
+"""Config flow for Tautulli."""
+from __future__ import annotations
+
+from typing import Any
+
+from pytautulli import (
+    PyTautulli,
+    PyTautulliException,
+    PyTautulliHostConfiguration,
+    exceptions,
+)
+import voluptuous as vol
+
+from homeassistant.components.sensor import _LOGGER
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_PATH,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_URL,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import (
+    DEFAULT_NAME,
+    DEFAULT_PATH,
+    DEFAULT_PORT,
+    DEFAULT_SSL,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+)
+
+
+class TautulliConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Tautulli."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
```

Comment:
Existing configuration support multiple servers (given it is using platform configuration right now). Any specific reason we don't support multiple now?

Reply:
There is a Plex server id that I believed to be a good unique id but @ludeeus disagreed. Honestly how many plex servers does a person need lol

Reply:
That ID is tied to a plex instance not tautulli instance.
If this was the Plex integration, that ID could be used, but it's not.

Reply:
If there is no unique ID, well then there is none 🤷